app/src/ui/app-error.tsx

```diff
@@ -239,6 +248,28 @@ export class AppError extends React.Component<IAppErrorProps, IAppErrorState> {
     )
   }
 
+  private renderRetryFooter(error: Error) {
```

Comment:
Was sort of surprised we didn't already have a means to have a retry-able error without making a custom error dialog. Seems like the retry clone stuff here could have been more generic.

Reply:
(ignore from first stab)